src/mudlet.pro

```diff
@@ -111,19 +111,23 @@ VERSION = 4.17.2
 
 # if you are distributing modified code, it would be useful if you
 # put something distinguishing into the MUDLET_VERSION_BUILD environment
-# variable to make identification of the used version simple
-# the qmake BUILD variable is NOT built-in
-BUILD_TEST = $$(MUDLET_VERSION_BUILD)
-isEmpty( BUILD_TEST ) {
+# variable (it should use '-' as the first character) to make identification of
+# the used version simple the qmake BUILD variable is NOT built-in
+BUILD = $$(MUDLET_VERSION_BUILD)
+isEmpty( BUILD ) {
 # Possible values are:
 # "-dev" for the development build
 # "-ptb" for the public test build
 # "" for the release build
-   BUILD_TEST = "-dev-"$${GIT_SHA1}
+   BUILD = "-dev-"$${GIT_SHA1}
 } else {
-   BUILD_TEST = $${GIT_SHA1}
+   BUILD = $${BUILD}-$${GIT_SHA1}
 }
-write_file( app-build.txt, BUILD_TEST )
+
+!build_pass:message("Value written to app-build.txt file: " $${BUILD})
+
+# This does append a line-feed to the file which might be problematic!
```

Comment:
I think this might have been causing other problems - but I've taken care to solve that now when the file is read.